lib/sign_up.rb

```diff
@@ -0,0 +1,56 @@
+require 'bcrypt'
+require_relative './database_connection'
+
+class ChitterSignUp
+
+  attr_reader :user_id, :name, :username, :email, :password
+
+  def initialize(user_id, name, username, email, password)
+    @user_id = user_id
+    @name = name
+    @username = username
+    @email = email
+    @password = password
+  end
+
+  def ==(other)
+    @user_id == other.user_id
+  end
+
+  def self.new_user(name, username, email, password)
```

Comment:
This probably ought to be self.create - again, this is only a convention, but generally speaking conventions should be followed!